docs/src/utils/makeData.tsx

```diff
@@ -33,8 +34,8 @@ export default function makeData<T>(lens: Array<number>, useRowActionColumn?: bo
             return {
                 ...newPerson(),
                 subRows: lens[depth + 1] ? makeDataLevel(depth + 1) : undefined,
-                actionLinks: useRowActionColumn ? (d % 2 === 0 ? actionLinks : null) : null,
-                actionButtonState: useRowActionColumn ? (d % 2 === 0 ? "disabled" : null) : null,
+                // actionLinks: useRowActionColumn ? (d % 2 === 0 ? actionLinks : null) : null,
+                // actionButtonState: useRowActionColumn ? (d % 2 === 0 ? "disabled" : null) : null,
```

Comment:
If you don't need this block of code you can just remove it instead of commenting it